posts/2022-09-19-db-sync.markdown

```diff
@@ -0,0 +1,33 @@
+# DBSync Update
```

Comment:
I wonder if it'd make sense to wrap lines to 80 characters.

Reply:
I've been (quite arbitrarily!) trying to keep lines to 100 characters, except when I have a link that's really long (or maybe it's in the CIPs where I hit that..)

Reply:
I changed this to max 100 chars/line, except for big links.